reactioncommerce/reaction
# refactor: remove addressBookRemove meteor method and rewrite with GraphQL

Resolves #5338  
Impact: **minor**  
Type: **feature|refactor**

## Issue
`addressBookRemove` still uses a meteor method to perform it's actions.

## Solution
Rewrite `addressBookRemove` mutation to use GraphQL instead of meteor method.
- Remove `addressBookRemove` meteor method from app

## Breaking changes
Nothing in core code. Any custom code calling the `accounts/addressBookRemove` Meteor method will break.

## Testing
1. Add an address to an account
1. Remove the address from the account
1. See that this works, with GQL now that the meteor method is removed
_This can be done through Example Storefront, with a logged in account_
